vignettes/malbacR_vignette_condensed_filtered.Rmd

```diff
@@ -103,34 +105,32 @@ amide_eigen <- bc_eigenMS(omicsData = pmart_amideFilt)
 # WaveICA2.0
 amide_wave <- bc_waveica(omicsData = pmart_amideFilt, injection_cname = "Injection_order",
                          alpha = 0, cutoff = 0.1, K = 10)
```

Comment:
The parameter names got updated, but weren't changed here
```suggestion
amide_wave <- bc_waveica(omicsData = pmart_amideFilt, injection_cname = "Injection_order",
                         version = "WaveICA2.0", alpha = 0, cutoff_injection = 0.1, K = 10)
```

Reply:
This also applies to the extended vignette as well, and the description paragraph will need to be modified as well.